src/v2/Apps/Settings/Routes/Purchases/Components/SettingsPurchasesRow.tsx

```diff
@@ -0,0 +1,351 @@
+import {
```

Comment:
All this code is largely the same, just cleaned up a bit. There's some issues with it we can tackle separately or ignore.